blog/2015-05-27-so-you-want-to-transform-part-1.adoc

```diff
@@ -0,0 +1,215 @@
+gained some cool new toys in the tooling and a big part of that is the
+Camel Editor (i.e. the "Fuse Tooling Routes Editor"). The Fuse
+Transformation tooling builds on that and offers:
+
+. the Fuse Transformation Editor, which offers visual editing of a Dozer
+configuration file
+
+. the New Fuse Transformation wizard, which creates a new Transformation
+file and integrates it as an endpoint in a Camel configuration
+
+. and the New Fuse Transformation Test wizard, which creates the outline
+for a new JUnit test you can run to test your transformation
+
+Though we're in Tech Preview for now, we have good collection of
+functionality already and we'll walk through some of that.
+
+== Where Can We Get These Toys?
+
+The new tooling appears in the JBoss Tools Integration Stack "Early
+Access" site, so be sure to keep up with that effort. (You can see
+Paul's latest post here: "JBoss Tools Integration Stack - Luna
+integration tooling for BRMS/BPMS, Data Virt, SOA 5.x + Early Access
+Fuse and SwitchYard."
+
+link:http://tools.jboss.org/blog/2015-04-16-JBTIS-421-Final.html[]
+
+Keep up to date with JBTIS to ensure you have the latest version of the
+Transformation tooling.
+
+== Let's Start With a Use Case - XML to JSON
+
+I'm going to cheat a little and start with the "Starter" project that we
+have created for the Transformation project. It includes some sample
+data to demonstrate transforming from XML to JSON. It includes a sample
+Camel configuration as well (in either Spring or Blueprint form) that
+provides a route that takes a file and routes it to another. In this
+case, we're going from a sales order in XML format (abc-order.xml) to a
+sales order in JSON format (xyz-order.json). This is a common
+integration case where you have to migrate one format to another.
+
+link:https://github.com/fusesource/fuseide/tree/master/examples/transformation/starter[]
+
+== But Wait! Didn't You Say Dozer Is Strictly Java-to-Java?
+
+Why yes, I did! But I also said that we have ways to create Java models
+from other formats. And in this first wave of Transformation
+functionality we offer built-in conversion from XML (XML schemas or
+instance documents) and JSON (JSON schemas or instance documents) to
+help get you that far. We will walk through each step of getting you
+from your sample data to a fully testable solution.
+
+So let's get started...
+
+== Launching the Wizard
+
+There are three main ways to launch the New Transformation Wizard in
+Eclipse...
+
+1.  From the top-level menu in the Eclipse workbench, use
+File->New->Other... (or Ctrl+N), then look for Fuse Tooling->Fuse
+Transformation
+2.  Right-click on a project in the Project Explorer, select
+New->Other... then look for Fuse Tooling->Fuse Transformation
+3.  Open a Camel file (Spring or Blueprint) and drag the
+Transformation->Data Transformation item from the palette onto the
+canvas (see below)
+
+image:images/transformation_tools/launching-the-wizard.png[image]
```

Comment:
not sure if we have a bug but for now use the absolute relative path to get the image to show up  right.

In this case:
image:/blog/images/transformation_tools/launching-the-wizard.png[]

And leave [] empty if you can't come up with an alternate descriptive text.